### PR TITLE
Support to get message exchange

### PR DESCRIPTION
**Pull request by prakashmrec**

### CODE REVIEW

File: src/main/resources/ready-api-rest-api-spec.json [hunk 1]
```diff
@@ -327,6 +327,92 @@
       }
     }
   },
+  "/v1/readyapi/executions/{executionID}/logs": {
```
Comment (olensmar) [inline src/main/resources/ready-api-rest-api-spec.json]: I think we can remove this file and generate directly from the definition on SwaggerHub

Comment (prakashmrec) [inline src/main/resources/ready-api-rest-api-spec.json]: Sure, one problem I have is that sometimes we need to work on different branches (specially when multuple people are working on it) and the SwaggerHub definition is compatible only with  one of them. But probably we can keep a short lived local copy of the SwaggerHub definition.

Comment (olensmar) [inline src/main/resources/ready-api-rest-api-spec.json]: Perhaps we can have temporary versions on SwaggerHub for those branches?

Comment (prakashmrec) [inline src/main/resources/ready-api-rest-api-spec.json]: Sounds good, How do we merge it back to main version? It should be possible do a diff and copy the changes to main version.

Comment (olensmar) [inline src/main/resources/ready-api-rest-api-spec.json]: yeah - that would be nice - i'm guessing its on the SH roadmap but we'll have to do it manually for now...

Comment (prakashmrec) [inline src/main/resources/ready-api-rest-api-spec.json]: ok, I will remove this file. thanks.

Comment (prakashmrec) [inline src/main/resources/ready-api-rest-api-spec.json]: Removed.

**Comment by olensmar** [comment]
I think this is a good start - but I think we should provide convenience methods to get request/response data for TestSteps - the use case that has been described is that users want to extract values from a response and use that in some way (without using property-transfers) - so methods to easily get response content and headers and extract values from them via json/x-path would be nice (my main concern with the latter is that it would introduce additional dependencies...). 

Perhaps we can merge this and I can create those methods as a separate PR?

**Comment by prakashmrec** [comment]
Sure, good point. Yes, separate PR would be good.

File: src/main/java/com/smartbear/readyapi/client/execution/SmartestApiWrapper.java [hunk 2]
```diff
@@ -17,5 +18,7 @@
```
Comment (olensmar) [inline src/main/java/com/smartbear/readyapi/client/execution/SmartestApiWrapper.java]: can we rename this class to simply "TestServerApi" ?

Comment (prakashmrec) [inline src/main/java/com/smartbear/readyapi/client/execution/SmartestApiWrapper.java]: sure, this was created when we were still trying few names for test server. I will rename.

Comment (olensmar) [inline src/main/java/com/smartbear/readyapi/client/execution/SmartestApiWrapper.java]: thanks :-)

Comment (prakashmrec) [inline src/main/java/com/smartbear/readyapi/client/execution/SmartestApiWrapper.java]: Done

**Comment by olensmar** [comment]
great - thank you!

**Pull request merged**